__tests__/utxo.test.js

```diff
@@ -0,0 +1,111 @@
+import wallet from "../src/wallet";
+import HathorWallet from "../src/new/wallet";
+import txHistoryFixture from "./__fixtures__/tx_history";
+import transaction from "../src/transaction";
+
+class FakeHathorWallet {
+  constructor() {
+    this.wallet = wallet;
+    this.getUtxos = HathorWallet.prototype.getUtxos.bind(this);
+    this.consolidateUtxos = HathorWallet.prototype.consolidateUtxos.bind(this);
+    this.sendManyOutputsTransaction = jest.fn(() => ({
+      success: true,
+      promise: Promise.resolve({ tx_id: "123" }),
+    }));
+  }
+  getTxHistory() {
+    return txHistoryFixture;
+  }
+}
+
+describe("UTXO Consolidation", () => {
+  let hathorWallet;
+  const destinationAddress = "WVGxdgZMHkWo2Hdrb1sEFedNdjTXzjvjPi";
+  beforeAll(() => {
+    transaction.updateMaxInputsConstant(255);
```

Comment:
Move `255` to a local constant in this file.

Reply:
Done